src/Microsoft.Restier.EntityFramework/Model/ModelMapper.cs

```diff
@@ -60,10 +61,10 @@ public ModelMapper(Type dbContextType)
             if (property != null)
             {
                 var type = property.PropertyType;
-                if (type.IsGenericType &&
-                    type.GetGenericTypeDefinition() == typeof(DbSet<>))
+                var parent = GetGenericParent(type);
```

Comment:
It is OK for me - C# is with single inheritance, and `GetInterfaces` returns all interfaces the class/interface implements, and inherits.  Please proceed with the change.  Thanks.